onboarding_project/squad_pantry_app/tests.py

```diff
@@ -1 +1,21 @@
 from django.test import TestCase
+from django.utils import timezone
+from squad_pantry_app.models import Order, SquadUser
+
+
+class OrderTestCase(TestCase):
+    def setUp(self):
+        self.user1 = SquadUser.objects.create(username="prakhar", password="testdb1", is_superuser=True, is_kitchen_staff=False)
+        self.user2 = SquadUser.objects.create(username="prakhar2", password="testdb2", is_superuser=True, is_kitchen_staff=False)
+
+    def test_cancel_order(self):
```

Comment:
Add  one more test case to see if the validation that only the user who put in the order can cancel the order.

Reply:
Have added that test.